### ISSUE TITLE
Devserver cannot serve static assets from parent or sibling directories

### ISSUE THREAD

**Issue by mrmeku** [bazelbuild/rules_typescript#165]
Consider a project with the following structure:
```
//apps/my-app-1/src/...
         my-app-2/src/...
//libs/my-common-asset/src/asset.png
```

We cannot serve the static asset out of the `//libs` directory since the devserver build rule is located within `//apps/APP_NAME/src:devserver`. This already affects .js files from `node_modules` (such as zone.js). We need a more generic solution than what we do with zone.js currently (which is copying the file to where the devserver build rule is).

**Comment by alexeagle** [comment]
Yeah in google3 there's a way to add more roots to ts_devserver, we should do the same

**Comment by gregmagolan** [comment]
Resolved with #177. Example usage of `additional_root_paths` in `ts_devserver` can be found here (https://github.com/alexeagle/angular-bazel-example/blob/master/src/BUILD.bazel#L32): 

```
ts_devserver(
    name = "devserver",
    # serve these files rooted at /
    additional_root_paths = [
        "npm/node_modules/zone.js/dist",
        "npm/node_modules/tslib",
        "npm/node_modules/@ngrx/store/bundles",
    ],
    # Start from the development version of the main
    entry_module = "angular_bazel_example/src/main.dev",
    scripts = [
        ":require.config.js",
    ],
    # This is the URL we'll point our <script> tag at
    serving_path = "/bundle.min.js",
    # Serve these files in addition to the JavaScript bundle
    static_files = [
        "@npm//node_modules/zone.js:dist/zone.min.js",
        "@npm//node_modules/tslib:tslib.js",
        "@npm//node_modules/@ngrx/store:bundles/store.umd.min.js",
        "index.html",
    ],
    # Tell Bazel to build the sources first
    deps = ["//src"],
)
```
